packages/scratch-gui/src/components/scratch-image/scratch-image.jsx

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
```

Comment:
Do scratchfoundation/scratch-editor#140 and scratchfoundation/scratch-editor#220 use the same commits for `ScratchImage`? Just nervous about trying to merge them both to `develop` at some point :sweat_smile: